src/sentry/issues/escalating.py

```diff
@@ -343,12 +343,20 @@ def manage_issue_states(
             add_group_to_inbox(group, GroupInboxReason.ONGOING, snooze_details)
             record_group_history(group, GroupHistoryStatus.ONGOING)
     elif group_inbox_reason == GroupInboxReason.UNIGNORED:
-        updated = Group.objects.filter(
+        group = Group.objects.filter(
             id=group.id, status__in=[GroupStatus.RESOLVED, GroupStatus.IGNORED]
-        ).update(status=GroupStatus.UNRESOLVED, substatus=GroupSubStatus.ONGOING)
+        )
+        if not group.exists():
+            return
+
+        group = group.first()
+        updated = False
+        is_new_group = group.first_seen > datetime.now(timezone.utc) - timedelta(days=3)
```

Comment:
Small thing, can we use this const for the `timedelta(days=3)` https://github.com/getsentry/sentry/blob/b277e025b34284021d40111db7fd6fee12a67e30/src/sentry/tasks/auto_ongoing_issues.py#L22